README.md

```diff
@@ -25,9 +22,8 @@ When using this together with `esp-println` make sure to use the same output kin
 | esp32h2           | Target ESP32-H2                                                      |
 | panic-handler     | Include a panic handler, will add `esp-println` as a dependency      |
 | exception-handler | Include an exception handler, will add `esp-println` as a dependency |
-| print-uart        | Use UART to print messages\*                                         |
-| print-jtag-serial | Use JTAG-Serial to print messages\*                                  |
-| defmt-espflash    | Use `defmt` logging to print messages\* (check [example](https://github.com/playfulFence/backtrace-defmt-example))                    |
+| println           | Use `esp-println` to print messages                                  |
+| defmt             | Use `defmt` logging to print messages\* (check [example](https://github.com/playfulFence/backtrace-defmt-example))                    |
```

Comment:
With these changes, the example is now outdated, shall we remove it or try to keep it up to date? Maybe just add a `println` and `defmt` example in this crate would be a better solution? As we can check those in CI